.github/workflows/readme_integration.yml

```diff
@@ -0,0 +1,34 @@
+name: Testing Haystack-Readme integration
+
+on:
+  push:
+    branches:
+      - readme_integration_api
```

Comment:
This should be `main` provided that the category IDs point to the "latest" version of the docs

Reply:
Yes of course